apps/www/src/registry/default/plate-ui/code-block-format-button.tsx

```diff
@@ -0,0 +1,31 @@
+'use client';
+
+import type { TElement } from '@udecode/plate';
+
+import { useCodeBlockFormat } from '@udecode/plate-code-block/react';
+import { BracesIcon } from 'lucide-react';
+
+import { Button } from './button';
+
+export function CodeBlockFormatButton({ element }: { element: TElement }) {
```

Comment:
Please move this into `code-block-element.tsx`, we try to avoid splitting small ui files